docs/_docs/1_1_quick_start_guide.md

```diff
@@ -117,7 +117,7 @@ df.write.format("hudi").
   options(getQuickstartWriteConfigs).
   option(PRECOMBINE_FIELD_OPT_KEY, "ts").
   option(RECORDKEY_FIELD_OPT_KEY, "uuid").
-  option(PARTITIONPATH_FIELD_OPT_KEY, "partitionpath").
+  option(PARTITIONPATH_FIELD_OPT_KEY, "partition_path").
```

Comment:
hi, would we keep the `partitionpath` as it is?

Reply:
Well in some places it said "partitionpath" and in some places it said "partitionPath". I do like snake case better for text names, but I'm open to any of the options as long as we're consistent.
For instance:
Line 89: spark.sql("select uuid, **partitionPath** from hudi_trips_snapshot").count()
Line 120: option(PARTITIONPATH_FIELD_OPT_KEY, "**partitionpath**").

Reply:
> Well in some places it said "partitionpath" and in some places it said "partitionPath". I do like snake case better for text names, but I'm open to any of the options as long as we're consistent.
> For instance:
> Line 89: spark.sql("select uuid, **partitionPath** from hudi_trips_snapshot").count()
> Line 120: option(PARTITIONPATH_FIELD_OPT_KEY, "**partitionpath**").

let's change to `partitionpath`, wdyt?